ppl/src/test/java/org/opensearch/sql/ppl/utils/PPLQueryDataAnonymizerTest.java

```diff
@@ -105,6 +105,19 @@ public void testEvalCommand() {
     assertEquals("source=t | eval r=abs(f)", anonymize("source=t | eval r=abs(f)"));
   }
 
+  @Test
+  public void testFillNullSameValue() {
+    assertEquals(
+        "source=t | fillnull with 0 in f1,f2", anonymize("source=t | fillnull with 0 in f1,f2"));
+  }
+
+  @Test
+  public void testFillNullVariousValues() {
+    assertEquals(
+        "source=t | fillnull using f1 = 0, f2 = -1",
```

Comment:
sounds this anonymize the expression?  
e.g. 
```
        "source=t | fillnull using f1 = ***, f2 = ***",
```

reference: `testAndExpression`

Reply:
I'm not sure what you mean here. Using a value of `***` would likely change the data type in this case since it looks like `f1` and `f2` are int values.

Reply:
The anonymizer should be removing all user-defined values and anonymizing the logger output. 
You can look at other examples in this test file for anonymized output.

Reply:
This is getting way off track. Only null values are replaced. This feature is not at all suited for anonymizing data (all non null values would be unchanged). If you want to anonymize data, you could use `eval` to create a new field and then `fields` to include the new field and ignore the old field.